package.json

```diff
@@ -303,7 +303,7 @@
         "@azure/arm-network": "^22.0.0",
         "fs-extra": "^8.1.0",
         "semver": "^5.7.0",
-        "vscode-azureextensionui": "^0.34.0",
+        "vscode-azureextensionui": "^0.35.0",
```

Comment:
Is there a reason you bumped ui for this PR?

Reply:
Was having issues with the back step so wanted to make sure that wasn't the problem.  I left it since I figured no harm in updating the UI package though